src/ape/cli/utils.py

```diff
@@ -10,10 +13,12 @@ class Abort(click.ClickException):
     useful for all user-facing errors.
     """
 
-    def __init__(self, message):
-        if not message.endswith("."):
-            message = f"{message}."
-
+    def __init__(self, message: Optional[str] = None):
+        caller = getframeinfo(stack()[1][0])
+        message = (
+            message
+            or f"Operation aborted in {caller.filename}::{caller.function} on line {caller.lineno}."
```

Comment:
Do we want to just always append `caller.filename` and `caller.function` to the message somewhere or maybe save it as a property of the exception so it can be debugged?

Reply:
If you are really trying to debug the stack, this info is probably already tracked, but I often feel like our errors don't have enough context for me to determine if it's a problem with something I wrote, or an actual bug in ape

Reply:
Have you tried `-v debug` ?